docs/src/user-manual/glossary.md

```diff
@@ -0,0 +1,368 @@
+bagging, compression, and storage, any AIPs created during ingest may also
+undergo any additional Post-ingest activities defined in the system. Examples of
+preservation processing workflow steps include file format characterization, and
+preservation normalization.
+
+## Preservation storage service
+
+A component of Enduro that acts as an interface between the **preservation
+engine** and whatever storage devices have been configured for the long-term
+preservation of **AIPs** produced by Enduro **workflows**. See:
+[Components](/src/user-manual/components.md).
+
+## Processing Information Package (PIP)
+
+A type of **package** derived from a **SIP** following all **ingest**
+activities, intended to be sent to the **preservation engine** for
+transformation into one or more **Archival Information Packages (AIPs)** for
+long-term preservation.
+
+In Enduro, a PIP is a transitional package state in the preservation
+**workflow** intended to standardize inputs to the preservation engine, and not
+a package type that Enduro operators will typically interact with directly.
+
+## Processing storage service
+
+A component of Enduro that acts as Enduro's storage back-end for any local
+**package** interactions that are needed as part of **ingest** and preservation
+**workflows**. See: [Components](/src/user-manual/components.md).
+
+## Producer
+
+Persons and/or organizations responsible for the submission of **packages**
+(**SIPs**) to be considered for long-term preservation by the preservation
+system and its operators.
+
+## Replication
+
+A **post-preservation** activity to produce duplicate copies (i.e. replicas) of
+a **package** such as an **AIP**, often in separate geo-redundant locations from
+the original, to mitigate risk and assist in recovery should any issue be found
+with the original package throughout its ongoing preservation.
+
+## Submission Information Package (SIP)
+
+A type of **package** submitted by a **producer** to be considered for long-term
+preservation. A SIP is also sometimes called a **transfer**.
+
+The term Submission Information Package was originally defined in the
+[OAIS](https://en.wikipedia.org/wiki/Open_Archival_Information_System) Reference
+Model created by the Consultative Committee for Space Data Systems (and since
+formally recognized as an international standard in ISO 14721).
+
+## System error
+
+A technical error in a **workflow** that is caused by a problem in one or more
+of the applications, networks, or connecting interfaces, typically requiring a
+system administrator or developer to resolve. See also:
+[Content error](#content-error).
+
+## Task
+
+An operation performed on a **file** or **package** in the context of a
+**workflow**. Also called an **activity**.
+
+## Transfer
+
+See: [Submission Information Package (SIP)](#submission-information-package-sip)
+
+## Watched directory
+
+A filesystem directory used that is configured with a small command-line utility
```

Comment:
I think this can be shortened to "A filesystem directory that is monitored for changes (e.g. adding, deleting, or renaming a file), and where such a change may trigger one or more subsequent actions (e.g. preservation processing)."

Reply:
updated!